46sshd/module-setup.sh

```diff
@@ -84,6 +84,12 @@ install() {
     echo systemd-tty-ask-password-agent >> "$initdir/root/.bash_history"
     chmod 600 "$initdir/root/.bash_history"
 
+    # sshd requires /var/log/lastlog for tracking login information
+    if [ ! -d "$initdir/var/log" ]; then
```

Comment:
I would remove this this conditional statement and unconditionally call `mkdir -p ...`.